src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -7,21 +7,58 @@
 using ALayoutDirection = Android.Views.LayoutDirection;
 using ATextDirection = Android.Views.TextDirection;
 using AView = Android.Views.View;
+using ViewHelper = Com.Microsoft.Maui.ViewHelper;
 
 namespace Microsoft.Maui
 {
 	public static partial class ViewExtensions
 	{
-		const int DefaultAutomationTagId = -1;
-		public static int AutomationTagId { get; set; } = DefaultAutomationTagId;
+		public static int AutomationTagId { get; set; } = Resource.Id.automation_tag_id;
+
+		public static void Initialize(this AView nativeView, IView view)
+		{
+			if (!view.Initialized)
+			{
+				var context = nativeView.Context!;
+				var pivotX = (float)(view.AnchorX * context.ToPixels(view.Frame.Width));
+				var pivotY = (float)(view.AnchorY * context.ToPixels(view.Frame.Height));
+
+				// NOTE: use named arguments for clarity
+				ViewHelper.Set(nativeView,
+					enabled: view.IsEnabled,
+					visibility: (int)view.Visibility.ToNativeVisibility(),
+					alpha: (float)view.Opacity,
+					translationX: context.ToPixels(view.TranslationY),
+					translationY: context.ToPixels(view.TranslationY),
+					scaleX: (float)(view.Scale * view.ScaleX),
+					scaleY: (float)(view.Scale * view.ScaleY),
+					rotation: (float)view.Rotation,
+					rotationX: (float)view.RotationX,
+					rotationY: (float)view.RotationY,
+					pivotX: pivotX,
+					pivotY: pivotY,
+					layoutDirection: (int)GetLayoutDirection(view),
+					automationTagId: AutomationTagId,
+					automationId: view.AutomationId,
+					minHeight: (int)view.MinimumHeight,
```

Comment:
I think those need to be converted from MAUI to Android pixels. Also, MAUI does it's own layout-ing so I don't think those should be set at all.